tasks/section1.yml

```diff
@@ -1240,7 +1240,7 @@ print $NF }'"
       - rule_1.7.1.3
 
 - name: "NOTSCORED | 1.7.1.4 | AUDIT | Ensure permissions on /etc/motd are configured"
-  command: /bin/true
+  shell: stat -L -c "%a %u %g" /etc/motd  | egrep "644 0 0"
   register: result
```

Comment:
Audit tasks should register verbose variable name that ends with `_audit`. As you are adding AUDIT checks please change `register: result` to `register: `*`descriptive_check_name_audit`*